warehouse/migrations/versions/4490777c984f_migrate_existing_data_for_release_is_.py

```diff
@@ -55,7 +55,7 @@ def upgrade():
                 """
             )
         )
-        conn.execute("COMMIT")
+        conn.execute(sa.text("COMMIT"))
```

Comment:
This surfaced during a `make initdb` vs running migrations. I chose to preserve the text vs `conn.commit()` since the function has other semantics applicable to the running connection.